docs/mirror_configuration.md

```diff
@@ -134,3 +134,28 @@ Example:
 [mirror]
 root_uri = https://example.com
 ```
+
+
+### diff-file
+
+The diff file is a string containing the filename to log the files that were downloaded during the mirror.
+This file can then be used to synchronize external disks or send the files through some other mechanism to offline systems.
```

Comment:
Can we update this with some examples. I'm guessing you're using rsync or some standard tool here? Noting that would be helpful to potential future users I feel.